Application/Telegram/Helpers/IdentifyUserLocation.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Telegram.Helpers
+{
+
+    public class Temperatures
+    {
+
+        [JsonProperty("address")]
+        public Address Address { get; set; }
+
+    }
+
+
+    public class Address
+    {
+        [JsonProperty("country")]
+        public string Country { get; set; }
+
+    }
+
+
+    public class IdentifyUserLocation
+    {
+        private readonly HttpClient _client = new HttpClient();
```

Comment:
it's bad practice to call new HttpClient manually, but I am ok to have it for internship, in real project you should inject IHttpClientFactory and call CreateClient from that

Reply:
I will change it and inject  IHttClientFactory